File: matrices/starsWithForLoop.cpp
#include<iostream>
using namespace std;
int main(){
	char stars[5][5] = {0};
	for(int i = 0; i < 5; i++){
		for(int j = 0; j < 5; j++){
			if(i == j || i + j == 4){
				stars[i][j] = '*';
			}
			cout << stars[i][j] << " ";
		}
		cout << endl;
	}
	return 0;
}

File: matrices/starsWithForLoop.java
package springIntro;

import java.util.*;

public class starsWithForLoop {

	public static void main(String[] args) {
		char stars[][] = new char[9][9];
		for(int i = 0; i < 9; i++) {
			for(int j = 0; j < 9; j++) {
				if(i == j || (i + j) == 8) {
					stars[i][j] = '*';
					System.out.printf("%c", stars[i][j]);
				}
				else {
					stars[i][j] = ' ';
					System.out.printf("%c", stars[i][j]);
				}
			}
			System.out.printf("\n");
				
		}
			
	}

}